packages/snfoundry/scripts-ts/deploy.ts

```diff
@@ -22,19 +22,20 @@ const deployScript = async (): Promise<void> => {
 
   const ethContract = new Contract(ethAbi, ethAddress, deployer);
 
-  const tx = await ethContract.invoke("transfer", [
-    diceGameAddr,
-    1000000000000000000n,
-  ]);
+  // 0.05 Eth
+  const ethAmount = 50000000000000000n;
+
+  const tx = await ethContract.invoke("transfer", [diceGameAddr, ethAmount]);
   // const receipt = await provider.waitForTransaction(tx.transaction_hash);
```

Comment:
In case uncomment L29 and we want to use `waitForTransaction` we need to use `provider` from `deploy-contract`, should we leave it imported or leave a comment about the correct import? it can be confused with the import of `provider` from `starknet`